addon/styles/addon.scss

```diff
@@ -1 +1,99 @@
 @import 'node_modules/ember-frost-theme/scss/frost-theme';
+
+$frost-login-width: 500px;
+$frost-login-separator-width: 5px;
+$frost-login-padding: 40px;
+
+.frost-login {
+  width:$frost-login-width;
+  margin: 0 auto;
+  min-width:$frost-login-width;
+}
+
+.frost-login .panel {
+  background-color: white;
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  box-shadow: 0 4px 35px 0 rgba($frost-color-grey-1, 0.15);
+}
+
+.frost-login .separator-vertical {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: $frost-login-separator-width;
+  height: 100%;
+  overflow: hidden;
+}
+
+.frost-login .logo {
+  margin: 30px 0 0 30px;
+  height: 60px;
+}
+
+.frost-login .product {
+  margin: 20px 0 0 40px;
+  font-size: $frost-font-l;
+  letter-spacing: 4px;
+  color: $frost-color-grey-4;
+
+  .version {
+    font-size: $frost-font-s;
+    letter-spacing: 0;
+    color: $frost-color-grey-6;
+  }
+}
+
+.frost-login .frost-bunsen-form {
+  margin: 40px 0 0 40px;
+  width: $frost-login-width - $frost-login-separator-width - $frost-login-padding*2;
+
+  .frost-field {
+    input {
+      width: 100%;
+    }
+
+    .left-label .required {
+      display: none;
```

Comment:
Honestly, I think the removal of this is silly - Brian is being inconsistent with the designs here.